src/react/components/searchField.js

```diff
@@ -0,0 +1,166 @@
+import React, { useState, createRef, useEffect } from "react";
+import { X, Search } from "react-hero-icon/solid";
```

Comment:
@nplowman this changed to `@heroicons/react` as it is the official icon npm package. https://www.npmjs.com/package/@heroicons/react

Reply:
Re: https://github.com/SU-SWS/decanter-react/pull/71/files